javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,37 @@
+const Male = "male";
+const Female = "female";
+let isFormal = true;
+
+function getFullname(firstName, lastName, isFormal, gender) {
+    if (firstName === ' ' || lastName === ' ' || firstName === null || lastName === null || firstName === undefined || lastName === undefined) {
+        console.log(`Please provide a valid firstname/lastname for booking`);
+    }
+    else if (isFormal !== undefined && typeof isFormal !== "boolean") {
+        console.log(`Formal type is not valid`);
+    }
+    else if (isFormal === true && gender === "male") {
+        console.log(`User full name with formalname is  is Lord  ${firstName} ${lastName}`);
+    }
+    else if (isFormal === true && gender === "female") {
+        console.log(`User full name with formal name is Lady  ${firstName} ${lastName}`);
+    }
+    else if (isFormal !== true && gender === "male") {
+        console.log(`User full name without formal name  is  ${firstName} ${lastName}`);
```

Comment:
There is no difference between line 19 and 22 and 25. it is always better to avoid repeating.